naucse/routes.py

```diff
@@ -182,6 +184,20 @@ def subpage_url(page_slug):
     kwargs.setdefault('title', page.title)
     kwargs.setdefault('content', content)
 
+    if (page.subpages != None and page.slug in page.subpages.keys()
+    and "prev" in page.subpages[page.slug].keys() and "next" in page.subpages[page.slug].keys()) :
+        prev_segments = page.subpages[page.slug]["prev"].split(':')
+        next_segments = page.subpages[page.slug]["next"].split(':')
```

Comment:
If I'm reading correctly, this way the title of the link can't have a colon in it. Is that a reasonable limitation?

The configuration already comes from YAML. Why invent a special format for this field?